src/Select.js

```diff
@@ -901,7 +901,8 @@ const Select = React.createClass({
 	},
 
 	renderClear () {
-		if (!this.props.clearable || (!this.props.value || this.props.value === 0) || (this.props.multi && !this.props.value.length) || this.props.disabled || this.props.isLoading) return;
+
+		if (!this.props.clearable || this.props.value == null || this.props.multi && !this.props.value.length || this.props.disabled || this.props.isLoading) return;
```

Comment:
:white_check_mark: Done
Replaced as proposed by `this.props.value === undefined || this.props.value === null`